task_list/app/views/tasks/index.html.erb

```diff
@@ -1,10 +1,38 @@
-<ul>
-   <% @tasks.each do |task| %>
-    <li>
-      <%= link_to task.name, "/tasks/#{task.id}" %>
-      <%= button_to "Delete?",{ action: "destroy", id: task.id }, method: :delete, data: { confirm: "Last chance...you sure?" }, class: "delete_button" %>
-    </li>
-    <% end %>
-</ul>
-
-<div><%= link_to "+Add a new task", "/tasks/new", class: "add_task"%></div>
+<form id="check_complete" action="/complete" method="put">
```

Comment:
I think because you're using `button_to` for the complete and delete buttons within this table, you actually don't need this outer form. Each `button_to` helper call will generate its own form just for that button.